### PR TITLE
Add upgrade info for kubecost, and update 1.3.3 RN

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-75774

https://jira.d2iq.com/browse/D2IQ-75885 

## Description of changes being made
Add known issues section with note that kubecost deployment _may_ fail for some customers if their provisioner doesn't allow storage size <1Gi, and an upgrading section in 1.3.3 (not yet released) referencing/repeating kubecost docs and recs on upgrades. I struggled a bit with where to put this info / how to write it so please feel free to rewrite/move things! And whether we need to dupe this in the konvoy RN, since technically kommander is released with konvoy so I guess this upgrade path would be upgrading konvoy versions ❓ 🤔 

When this reviewed, will apply same RN updates over to 1.4 RN

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the upgrade instructions below.
+
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md):
+
+**If you are upgrading an existing version of Kommander**
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]: This info should go into an upgrade document, not the release notes.

**Review by cmastr — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 1 above)
Comment (gracedo) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]: Where should the upgrade doc go?

Comment (cmastr) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]: In the upgrade docs. Or maybe in the Kubecost section?

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      - We suggest you delete the old PV, then upgrade Kommander.
+    - If you cannot safely delete the PV storing your configs and configure them on a new PV:
+      - If you are not on a regional cluster, we recommend that you provision a second PV by setting `kubecost.cost-analyzer.persistentVolume.dbPVEnabled=true` in your cluster.yaml under Kommander values.
+      - If you are on a regional cluster,  we recommend that you provision a second PV using a topology-aware storage class ([more info](https://kubernetes.io/blog/2018/10/11/topology-aware-volume-provisioning-in-kubernetes/#getting-started)). You can set this disk’s storage class by setting `kubecost.cost-analyzer.persistentVolume.dbStorageClass=<your-topology-aware-storage-class-name>`.
+
+# Bug fixes
+
+- UI: Resolved kubecost performance issue.
+- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See Upgrading above if upgrading from a previous version. (COPS-6937)
+
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:42]:
```suggestion
```

File: pages/dkp/kommander/1.3/release-notes/1.3.0/index.md [hunk 3]
```diff
@@ -26,7 +26,9 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.19.x  |
 | **Default**        | 1.19.7  |
 
-# Known issue
+# Known issues
+
+In Kommander 1.3.2 and below, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that requiire a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.0/index.md:31]: FYI - just pushing this change - if you have big changes you'll need to pull first @gracedo 

```suggestion
In Kommander 1.3.2 and below, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:22]: We don't need this section, because this IS the upgrade path/fixed version.

That being said, it might just be nice to have this line edited, because people might just land on this page not knowing for sure what to do with their Kubecost business.

So, I don't know? Whatever you all think is best @gracedo and @cmastr @russurquhart1 @JohnB64-JBV 
We could probably wipe this section for 1.3.3 or just apply the change below. Or maybe not - after re-reading it, it is pretty clear. Either way.

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the upgrade instructions below.
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:24]:
```suggestion
In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to this version (Kommander v1.3.3) following the upgrade instructions below.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md):
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:27]:
```suggestion
The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md).
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md).
+
+**If you are upgrading an existing version of Kommander**
+  - If your provisioner does supports volume expansion, we will automatically resize you to a 32GB disk in an upgrade.
+  - If your provisioner does not support volume expansion:
+    - If you have not added or modified Kubecost configuration from the settings page on the Kubecost frontend, you can safely delete the PV and upgrade.
+      - We suggest you delete the old PV, then upgrade Kommander.
+    - If you cannot safely delete the PV storing your configs and configure them on a new PV:
+      - If you are not on a regional cluster, we recommend that you provision a second PV by setting `kubecost.cost-analyzer.persistentVolume.dbPVEnabled=true` in your cluster.yaml under Kommander values.
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:35]:
```suggestion
      - If you are not on a regional cluster, we recommend that you provision a second PV by setting `kubecost.cost-analyzer.persistentVolume.dbPVEnabled=true` in your `cluster.yaml` under the Kommander addon values.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+      - If you are on a regional cluster,  we recommend that you provision a second PV using a topology-aware storage class ([more info](https://kubernetes.io/blog/2018/10/11/topology-aware-volume-provisioning-in-kubernetes/#getting-started)). You can set this disk’s storage class by setting `kubecost.cost-analyzer.persistentVolume.dbStorageClass=<your-topology-aware-storage-class-name>`.
+
+# Bug fixes
+
+- UI: Resolved kubecost performance issue.
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:40]:
```suggestion
- UI: Resolved Kubecost performance issue.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See Upgrading above if upgrading from a previous version. (COPS-6937)
```
Comment (msdolbey) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:41]:
```suggestion
- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Upgrading above](#upgrading) if upgrading from a previous version. (COPS-6937)
```

**Review by msdolbey — APPROVED** [review]
👍 
Kubecost for Life

File: pages/dkp/kommander/1.3/release-notes/1.3.0/index.md [hunk 10]
```diff
@@ -26,7 +26,9 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.19.x  |
 | **Default**        | 1.19.7  |
 
-# Known issue
+# Known issues
+
+In Kommander 1.3.2 and below, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.0/index.md:31]:
```suggestion
In Kommander 1.3.2 and below, the Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 11]
```diff
@@ -26,13 +26,18 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.19.x  |
 | **Default**        | 1.18.10  |
 
+# Known issues
+
+Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:31]:
```suggestion
The Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md [hunk 12]
```diff
@@ -19,6 +19,10 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 # Release Summary
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
 
+# Known issues
+
+Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:24]:
```suggestion
The Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 5 above)
Comment (cneth) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:24]: Can we also get the article before Kubecost in here, please?

```suggestion
In versions prior to 1.3.3, the Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to this version (Kommander v1.3.3) following the upgrade instructions below.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+# Known issues
+
+In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the upgrade instructions below.
+
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md).
+
+**If you are upgrading an existing version of Kommander**
+  - If your provisioner does supports volume expansion, we will automatically resize you to a 32GB disk in an upgrade.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:30]:
```suggestion
  - If your provisioner supports volume expansion, the Kubecost PV will be automatically resized to 32GB during the upgrade.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 7 above)
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:35]: I'm having a lot of troubles following the bullets here, all the negative conditions make it very difficult to understand.

It seems like there are two major cases here:

1. You have not modified the default configs, in which case you should just delete the PV and then upgrade.  (Where do we explain how to find the right volume?)

If this is correct, this sentence seems like it would cover it:
   
    - If you have not added or modified the Kubecost configuration using the settings page on the Kubecost frontend, you can safely delete the PV and upgrade.  We suggest you delete the old PV first, and then upgrade Kommander.   (But - we need to add to this HOW to find the right PV.  Also - what happens if you don't delete the old PV before upgrade?)

2. You've modified the configs.   This is where it gets hard to follow what to do.   Is it possible to backup/restore them to a new PV?  If so, how? If not, what happens to my settings, do I have to redo them? 

I think we should write out all the scenarios here in long form and *then* try to condense it down to a bullet list.

Finally, it seems that Kubecost now has support for topology aware storage, but we only recommend using that if you have already customized the Kubecost settings and only for a 'second' PV?   The mention of a second PV is confusing to me.  

Also, this seems like a separate, new feature, which is independent of this one particular upgrade scenario.   Is this something I can use with a new installation?  If so, how?   Finally, let's make sure we use the same terminology as the K8S reference page here.  'Regional Cluster' isn't a term used anywhere else in our docs.  It seems like we mean "multi-zone cluster"?

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In Kommander 1.3.2 and earlier versions, Kubecost cost-analyzer requests a PVC of size 0.2Gi by default. This can cause issues with provisioners requiring a minimum storage size of 1Gi. Upgrade to version 1.3.3 to resolve this issue. Refer [here](https://github.com/kubecost/docs/blob/master/storage.md) for more information. 
+
+# Bug fixes
+
+- UI: Resolved Kubecost performance issue.
+- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Upgrading](#upgrading) above if upgrading from a previous version. (COPS-6937)
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]:
```suggestion
- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Known issues](#known-issues) above if upgrading from a previous version. (COPS-6937)
```

Should this be pointing to Known Issues since Upgrading was removed? Or is there going to be another page where the upgrade docs will be moved to?

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 4 above)
Comment (cmastr) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:22]: yeah I think we should add SOMETHING to release notes. We could add it as a fix, or we can add it as a known issue with a workaround.

**Review by russurquhart1 — APPROVED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 14 above)
Comment (russurquhart1) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]: Good point.  I'll fix that!

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In Kommander 1.3.2 and earlier versions, the Kubecost cost-analyzer requests a PVC of size 0.2Gi by default. This can cause issues with provisioners requiring a minimum storage size of 1Gi. Upgrade to version 1.3.3 to resolve this issue. Refer [here](https://github.com/kubecost/docs/blob/master/storage.md) for more information. 
+
+# Bug fixes
+
+- UI: Resolved Kubecost performance issue.
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:28]:
```suggestion
- UI: Resolved Kubecost performance issue. (COPS-6895)
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Kown issues](#known-issues) above if upgrading from a previous version. (COPS-6937)
+
+
+## Component versions
+
+- Addon: 1.3.3-4
+- Chart: 0.15.12
+- kommander-federation (yakcl): 0.8.11
+- kommander-licensing (yakcl): 0.8.11
+- UI: 6.91.3
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:38]:
```suggestion
- Addon: 1.3.3-5
- Chart: 0.15.13
- kommander-federation (yakcl): 0.8.12
- kommander-licensing (yakcl): 0.8.12
- UI: 6.91.8
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+- kommander-karma: 0.3.12
+- kubeaddons-catalog: 0.1.15
+- kommander-thanos: 0.1.16
+- kubecost: 0.11.0
+- grafana: 6.6.0
+- karma: 0.70
+- thanos: 0.10.1
+- cost-analyzer: 1.79.1
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:46]:
```suggestion
- cost-analyzer: 1.81.1
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.3
+title: Release Notes Kommander 1.3.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.3.3
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.3 was released on x, May 2021.**
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:11]:
```suggestion
**D2iQ&reg; Kommander&reg; version 1.3.3 was released on 9, June 2021.**
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In Kommander 1.3.2 and earlier versions, the Kubecost cost-analyzer requests a PVC of size 0.2Gi by default. This can cause issues with provisioners requiring a minimum storage size of 1Gi. Upgrade to version 1.3.3 to resolve this issue. Refer [here](https://github.com/kubecost/docs/blob/master/storage.md) for more information. 
+
+# Bug fixes
+
+- UI: Resolved Kubecost performance issue.
+- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Kown issues](#known-issues) above if upgrading from a previous version. (COPS-6937)
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:29]:
```suggestion
- Increased the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Known issues](#known-issues) above if upgrading from a previous version. (COPS-6937)
```

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+
+- Addon: 1.3.3-4
+- Chart: 0.15.12
+- kommander-federation (yakcl): 0.8.11
+- kommander-licensing (yakcl): 0.8.11
+- UI: 6.91.3
+- kommander-karma: 0.3.12
+- kubeaddons-catalog: 0.1.15
+- kommander-thanos: 0.1.16
+- kubecost: 0.11.0
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:42]:
```suggestion
- kubecost: 0.13.0
```

**Review by cneth — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Review by Colin1841 — COMMENTED** [review]
Review complete - 2 very minor changes.
